pkg/devfile/adapters/common/command_simple.go

```diff
@@ -28,7 +28,10 @@ func newSimpleCommand(command devfilev1.Command, executor commandExecutor) (comm
 
 	// deal with environment variables
 	var setEnvVariable, cmdLine string
-	for _, envVar := range exe.Env {
+	for i, envVar := range exe.Env {
```

Comment:
should we make a small util function to do this?

Reply:
+1, we're repeating this at multiple places. We should create a function for this.

Reply:
Fixed